src/common/Microsoft.DotNet.UpgradeAssistant.Abstractions/IPackageLoader.cs

```diff
@@ -11,7 +11,7 @@ public interface IPackageLoader
     {
         IEnumerable<string> PackageSources { get; }
 
-        Task<bool> DoesPackageSupportTargetFrameworkAsync(NuGetReference packageReference, string cachePath, TargetFrameworkMoniker targetFramework, CancellationToken token);
+        Task<bool> DoesPackageSupportTargetFrameworkAsync(NuGetReference packageReference, string cachePath, IEnumerable<TargetFrameworkMoniker> targetFrameworks, CancellationToken token);
```

Comment:
Nit-pick, we should change the name to DoesPackageSupportTargetFramework**s**Async since it takes multiple target frameworks now.